packages/main/src/Menu.ts

```diff
@@ -542,9 +546,18 @@ class Menu extends UI5Element {
 		this._parentItemsStack.push(item);
 	}
 
+	_onfocusin(e: FocusEvent): void {
+		const target = e.target as HTMLElement;
+		const menuListItem = target.hasAttribute("ui5-menu-li")
+			? target as MenuListItem
+			: (target.getRootNode() as ShadowRoot).host as MenuListItem;
+		const item = menuListItem.associatedItem as MenuItem;
+		const mainMenu = this._findMainMenu(item);
+		mainMenu?.fireEvent("ui5-item-focus", { ref: menuListItem, item });
```

Comment:
fireEvent("item-focus")

You don't have to set the prefix manually, this is done by the framework. With the current code it will fire 2 events: "ui5-item-focus" and "ui5-ui5-item-focus"

Reply:
Understood, done